dev/tasks/docker-tests/github.linux.yml

```diff
@@ -45,7 +45,7 @@ jobs:
             {{ image }} \
             {{ command|default("") }}
 
-    {% if '-r' in image %}
```

Comment:
Thanks for catching this — IIRC the original change from `-r-` to `-r` was to catch if we were using the `ubuntu-r` image. It looks like we don't have any nightlies that use that directly currently (they all use ubuntu-r-only-r). 

Would it be possible to do `if '-r-' in image | 'ubuntu-r' in image`?

Reply:
Or maybe that's YAGNI (but I do remember being confused why I wasn't getting the output when trying to use `ubuntu-r` in crossbow)

Reply:
Or can we use more verbose ID for R such as `rstats` and `rlang`?
It seems that `r` is too short to identify...

Reply:
Yeah, absolutely that would work too (I would probably vote for `rlang` of those two).

Reply:
I reconsider this.

How about "doing nothing" for these R related steps in non R related jobs instead of hide these steps by `{% if ... %}`?
(See https://github.com/apache/arrow/pull/12525/files for implementation.)

We don't need to care about docker-compose service name with this approach.

Reply:
I didn't use `cat file || true` approach because I want to avoid showing "cat: file: No such file or directory" message in CI log.